examples/ScatterRun/par_files/run_reconstruction.par

```diff
@@ -0,0 +1,32 @@
+Reconstruction :=
+reconstruction method := OSMAPOSL
+OSMAPOSLParameters :=
+
+objective function type:= PoissonLogLikelihoodWithLinearModelForMeanAndProjData
+PoissonLogLikelihoodWithLinearModelForMeanAndProjData Parameters:=
+maximum absolute segment number to process := -1
+
+projector pair type := Matrix
+  Projector Pair Using Matrix Parameters :=
+  Matrix type := Ray Tracing
+  Ray tracing matrix parameters :=
+   number of rays in tangential direction to trace for each bin:= 1
+  End Ray tracing matrix parameters :=
+  End Projector Pair Using Matrix Parameters :=
+
+recompute sensitivity :=1 
+zoom := 1
+
+end PoissonLogLikelihoodWithLinearModelForMeanAndProjData Parameters:=
+
+enforce initial positivity condition:=1
+
+number of subsets:= 1
```

Comment:
these are too low. let's not give them the impression that this would work

Reply:
Hmm, this is for the scatter estimation not the final reconstruction. It should work.